From b1f4747bcfeaa9962390b792e8ee15804cec2876 Mon Sep 17 00:00:00 2001
From: Alex Deucher <alexander.deucher@amd.com>
Date: Mon, 21 Dec 2015 17:07:40 -0500
Subject: [PATCH 0849/1565] amd/powerplay: disable powerplay by default
 initially

Hopefully we can enable this by default once we get more
upstream feedback on stability, etc.

Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_powerplay.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_powerplay.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_powerplay.c
index ddb90eb..5ee9a06 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_powerplay.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_powerplay.c
@@ -101,7 +101,7 @@ static int amdgpu_pp_early_init(void *handle)
 	switch (adev->asic_type) {
 		case CHIP_TONGA:
 		case CHIP_FIJI:
-			adev->pp_enabled = (amdgpu_powerplay == 0) ? false : true;
+			adev->pp_enabled = (amdgpu_powerplay > 0) ? true : false;
 			break;
 		default:
 			adev->pp_enabled = (amdgpu_powerplay > 0) ? true : false;
-- 
1.9.1

